tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "allowJs": true,
+        "esModuleInterop": true,
+        "forceConsistentCasingInFileNames": true,
+        "lib": ["es2021", "dom"],
```

Comment:
using `es2021` so we can use some handy string methods like `.startsWith` and `endsWith`